dashboard/test/fixtures/rollup_config.json

```diff
@@ -1,13 +1,25 @@
 {
```

Comment:
same change as non-test version of this file to use categories for rollup questions